src/libstd/macros.rs

```diff
@@ -315,8 +315,8 @@ macro_rules! eprintln {
 /// file and line whenever it's reached.
 ///
 /// [stderr]: https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)
-/// [debug-log]: https://docs.rs/log/*/log/macro.debug.html
-/// [log]: https://docs.rs/log/
+/// [`debug`]: https://docs.rs/log/*/log/macro.debug.html
```

Comment:
Missing exclamation `!`.

Reply:
Pffff..... I feel annoyed with myself that I forget such simple things...

Reply:
No worry! You're doing really well.

Reply:
Thanks! Truly trying my best here :)